api/login.ts

```diff
@@ -11,7 +11,7 @@ import { parseInput } from '../api-lib/signature';
 
 Settings.defaultZone = 'utc';
 
-const allowedDomains = (process.env.SIWE_ALLOWED_DOMAINS)?.split(",") || ["localhost:3000"];
+const allowedDomains = (process.env.SIWE_ALLOWED_DOMAINS)?.split(",").filter(item => item !== "") || ["localhost:3000"];
```

Comment:
do you need more than one domain in the config file? depending on contract vs EOA account verification, message.verify might check eip-1271 (requires Ethereum lookup). if only one domain is needed, i'd prefer removing the for loop for efficiency reasons. another option would be not checking for the domain in message.verify itself but instead checking manually after.

Reply:
I included the option for multiple domains because of this comment @crabsinger made here https://github.com/coordinape/coordinape/pull/967#discussion_r887215293

But good point regarding the domain verification, I was under the impression it needed a `domain` but if it doesn't I'll refactor to check without looping the whole `try`

Reply:
@awoie i want our server, which is validating these signatures, to reject signatures that originate from random sites that could be used for phishing. since we may want multiple domains to be valid originations, we need to support a number of domains. Thoughts on this? Right now, I'd prfer if we can implement this without querying an Ethereum RPC, if possible.